packages/template-retail-react-app/app/pages/checkout/partials/shipping-address.jsx

```diff
@@ -21,6 +21,8 @@ import {
 } from '@salesforce/commerce-sdk-react'
 import {useCurrentCustomer} from '@salesforce/retail-react-app/app/hooks/use-current-customer'
 import {useCurrentBasket} from '@salesforce/retail-react-app/app/hooks/use-current-basket'
+import ShippingMultiAddress from '@salesforce/retail-react-app/app/pages/checkout/partials/shipping-multi-address'
+import {MULTISHIP_IS_ENABLED} from '@salesforce/retail-react-app/app/constants'
```

Comment:
New MultiShip component

Reply:
We are having BOPIS toggle feature. If it is working with BOPIS, we should think of how to consolidate these two toggles together.

Reply:
@shauryemahajanSF Do you know this?